pysmt/walkers/generic.py

```diff
@@ -50,6 +50,34 @@ def __init__(self, env=None):
         self.functions[op.ITE] = self.walk_ite
         self.functions[op.TOREAL] = self.walk_toreal
 
+        self.functions[op.BV_CONSTANT] = self.walk_bv_constant
+        self.functions[op.BV_CONCAT] = self.walk_bv_concat
+        self.functions[op.BV_EXTRACT] = self.walk_bv_extract
+        self.functions[op.BV_NOT] = self.walk_bv_not
+        self.functions[op.BV_AND] = self.walk_bv_and
+        self.functions[op.BV_OR] = self.walk_bv_or
+        self.functions[op.BV_XOR] = self.walk_bv_xor
+        self.functions[op.BV_ULT] = self.walk_bv_ult
+        self.functions[op.BV_ULE] = self.walk_bv_ule
+#        self.functions[op.BV_SLT] = self.walk_bv_slt
```

Comment:
Signed operators are not part of the minimal BV definition. However, they are usually implemented as first-class citizens. This was here as reminder for future work.
We can either:
- Remove them
- Implement them